### ISSUE TITLE
Conformance test #2 has no output specified in the workflow

### ISSUE THREAD

**Issue by prismofeverything** [common-workflow-language/common-workflow-language#327]
Working through the conformance tests, I get to #2 and see this line:

```
outputs: []
```

Yet the test expects output. How is this possible? The workflow calls something that creates output, but how can anything downstream of this use it if it is not specified?

I have added to this output field and got the test to pass:

```
outputs: 
  - id: args
    type:
      type: array
      items: string
```

**Comment by StarvingMarvin** [comment]
Yup, this is a long standing problem, basically the same issue as #262. We will submit a pull request probably this week (certainly before London meetup). @sinisa88, @simonovic86

**Comment by StarvingMarvin** [comment]
@mr-c, @prismofeverything, this got fixed with #334
